datawig/_hpo.py

```diff
@@ -65,12 +67,12 @@ def __init__(self):
         default_hps = dict()
         default_hps['global'] = {}
         default_hps['global']['learning_rate'] = [3e-4]
-        default_hps['global']['weight_decay'] = [0, 1e-7]
-        default_hps['global']['num_epochs'] = [100]
+        default_hps['global']['weight_decay'] = [1e-7]
+        default_hps['global']['num_epochs'] = [10]
         default_hps['global']['patience'] = [5]
-        default_hps['global']['batch_size'] = [16]
-        default_hps['global']['final_fc_hidden_units'] = [[], [100]]
-        default_hps['global']['concat_columns'] = [True, False]
+        default_hps['global']['batch_size'] = [128]
```

Comment:
In my experience, batch size >16 hampers performance for any non-convex models. (Although it buys us much speed).